keps/sig-storage/20190129-csi-migration.md

```diff
@@ -123,6 +127,57 @@ The detailed design was originally implemented as a [design proposal](https://gi
 
 * All volume operation paths covered by Migration Shim in Beta for >= 1 quarter without significant issues
 
+## Test Plan
+
+### Per-driver migration testing
+
+We will require each plugin/driver provider to set up public CI to run all existing
+in-tree plugin driver tests for their migrated driver.
+
+If migration is on for that plugin, we will add checks in the test framework
+that inspects CSI Drivers metrics to make sure that the driver is servicing the
```

Comment:
I think this is describing a behavior of the k8s e2e test suite, but the wording is ambiguous. "we will" could refer to we, the humans talking about this.

If we were only describing test behavior in this section of the doc, it would not be ambiguous, but there are definitely parts of this section referring to things that humans will do (like setting up CI pipelines).